parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -51,7 +51,9 @@ installDeps() {
     fi
 
     if [ "${OSVERSION}" = "22.04" ] || [ "${OSVERSION}" = "24.04" ]; then
-        pkg_list+=("aznfs=0.3.15")
```

Comment:
Was there a reason why this was not added to components.json?

Reply:
there are two versions for different purposes now, finally the two versions would be combined, let's use the spefic version for now